plugins/modules/aci_syslog_group.py

```diff
@@ -239,24 +286,28 @@
 
 from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec
 from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.constants import SYSLOG_FORMATS
 
 
 def main():
     argument_spec = aci_argument_spec()
     argument_spec.update(aci_annotation_spec())
     argument_spec.update(
         name=dict(type="str", aliases=["syslog_group", "syslog_group_name"]),
-        format=dict(type="str", choices=["aci", "nxos"]),
+        format=dict(type="str", choices=["aci", "nxos", "rfc5424-ts", "enhanced_log"]),
+        local_file_log_format=dict(type="str", choices=["aci", "nxos", "rfc5424-ts", "enhanced_log"]),
+        console_log_format=dict(type="str", choices=["aci", "nxos", "rfc5424-ts", "enhanced_log"]),
```

Comment:
why do you not use the constant here?

```suggestion
        format=dict(type="str", choices=list(SYSLOG_FORMATS)),
        local_file_log_format=dict(type="str", choices=list(SYSLOG_FORMATS)),
        console_log_format=dict(type="str", choices=list(SYSLOG_FORMATS)),
```

Reply:
Sorry, I forgot about that, please let me know if there are any documentation changes I need to make for this.